From ccf9e82098714b102a276421975ea416853a73f0 Mon Sep 17 00:00:00 2001
From: Wallace Wang <r59996@freescale.com>
Date: Wed, 29 Oct 2008 16:13:27 +0800
Subject: [PATCH] ENGR00096982-2 MX51: Move spdif gpio configuration to spdif driver

The gpio can be dynamically requested/freed in driver module

Signed-off-by: Wallace Wang <r59996@freescale.com>
---
 arch/arm/mach-mx51/devices.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-mx51/devices.c b/arch/arm/mach-mx51/devices.c
index 8897e58..d5bd25b 100644
--- a/arch/arm/mach-mx51/devices.c
+++ b/arch/arm/mach-mx51/devices.c
@@ -29,8 +29,6 @@
 #include "sdma_script_code.h"
 #include <asm/arch/mxc_scc2_driver.h>
 
-extern void gpio_spdif_active(void);
-
 void mxc_sdma_get_script_info(sdma_script_start_addrs * sdma_script_addr)
 {
 	/* AP<->BP */
@@ -829,7 +827,6 @@ static struct platform_device mxc_alsa_spdif_device = {
 
 static inline void mxc_init_spdif(void)
 {
-	gpio_spdif_active();
 	mxc_spdif_data.spdif_core_clk = clk_get(NULL, "spdif_xtal_clk");
 	clk_put(mxc_spdif_data.spdif_core_clk);
 	platform_device_register(&mxc_alsa_spdif_device);
-- 
1.5.4.4

